src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -118,6 +116,20 @@ public virtual bool CanValidateToken
             get { return true; }
         }
 
+        private int CountTokenSegments(string token)
+        {
+            // Set the maximum number of segments to MaxJwtSegmentCount + 1. This controls the number of splits and allows detecting the number of segments is too large.
```

Comment:
This comment addresses the line that you are replacing with CountTokenSegments() method.
Can you update the comment to reflect your new logic?
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/18ebb69783db82c1911f38c75311aa33425f1ee8/src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs#L102

Reply:
Fixed.